### ISSUE TITLE
Remove component Switch and replace with the PF4 component

### ISSUE THREAD

**Issue by rszwajko** [oVirt/ovirt-web-ui#1408]
Feature enhancement request:

Follow-up issue after #1396 

Existing [custom Switch](https://github.com/oVirt/ovirt-web-ui/blob/master/src/components/Switch.js) does not bring much value and can be replaced by PatternFly Switch component.

**Comment by rszwajko** [comment]
The main functionality provided by the  custom Switch was translating the ON/OFF messages.

@sjd78 @sgratch 
Do we want to stop translating ON/OFF labels? Or just inline the `onText`, `offText` props?

FIY the current functionality has 2 serious bugs:
1. OFF message is not translated due to typo (`offTest`  instead of `offText`)
2. widget is broken on VM Detail screen if translated label is longer then English  (caused by forcing `handleWidth={30}`)

**Example** (the upper switch is not translated to show the difference):
![switch_off](https://user-images.githubusercontent.com/64194103/116688038-66df8a00-a9b6-11eb-8622-55c2c5bf368a.png)

**Comment by sjd78** [comment]
@rszwajko, @sgratch - I vote that we just follow the current PF switch guidelines: https://www.patternfly.org/v4/components/switch

Basically always show the slider and enable/disable it based on the edit state.

I don't see a need for a text label driven by the state since we have decent field labels to begin with.  If we really want to drive an on/off message, that should customized the the field.
